pinot-spi/src/main/java/org/apache/pinot/spi/accounting/ThreadResourceUsageAccountant.java

```diff
@@ -50,16 +50,26 @@ void createExecutionContext(String queryId, int taskId, ThreadExecutionContext.T
    * @param taskId a unique task id
    * @param taskType the type of the task - SSE or MSE
    */
+  @Deprecated
```

Comment:
If `workloadName` can be null, can this be not deprecated ? It is a useful short form for when `workloadName` is not required.

Reply:
workloadName can never be null. QueryOptionUtils::getWorkloadName always returns a default workload name.